### ISSUE TITLE
FeatureRequest: can you add Display for BrowserKind 

### ISSUE THREAD

**Issue by felipetesc** [JAD3N/browsers#1]
Since we cannot add a display impl for external enums, I wonder if you can add the following to the source, so we can print the browser as a string if we want to println the possibilities while working with cli: 

```rust


impl fmt::Display for BrowserKind {
    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
        match self {
            BrowserKind::Chrome => write!(f, "Chrome"),
            BrowserKind::Chromium => write!(f, "Chromium"),
            BrowserKind::Firefox => write!(f, "Firefox"),
            BrowserKind::Safari => write!(f, "Safari"),
            BrowserKind::Brave => write!(f, "Brave"),
            BrowserKind::Opera => write!(f, "Opera"),
            
            }
    }
}
```

Thanks !!!

**Comment by JAD3N** [comment]
Added in by https://github.com/JAD3N/browsers/commit/3f9f3f5cf73c6d79ff023c1d28afe477c02496d8, and published in a new patch version (v0.1.1).

**Comment by felipetesc** [comment]
> Added in by [3f9f3f5](https://github.com/JAD3N/browsers/commit/3f9f3f5cf73c6d79ff023c1d28afe477c02496d8), and published in a new patch version (v0.1.1).

That was fast!
